src/pages/workflows/workflow/WorkflowDetails.js

```diff
@@ -101,18 +102,16 @@ const SnapshotWrapper = ({ namespace, name, snapshotId, tabName, children }) =>
       displayNames: { configs: 'configurations' },
       getHref: currentTab => Nav.getLink(`workflow-${currentTab}`, { namespace, name, snapshotId: selectedSnapshot })
     }, [
```

Comment:
With my change to the TabBar, the children inside of `IdContainer` were not getting rendered. `IdContainer` is now deprecated, so I changed the code to use `useUniqueId` and it fixed the issue.